### PR TITLE
calculator app milestone 1

### PR DESCRIPTION
**Pull request by ashraffares**
- Create React app with the create-react-app command.
- Remove unnecessary files.
- Add React linters.
- create Heroku app

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "name": "calculator",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
```
Comment (rindrajosia) [inline package.json:6]: - [ ] [REQUIRED] [big.js](https://github.com/MikeMcl/big.js/) added to package.json. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/big.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))
 - [ ] [REQUIRED] [prop-types](https://www.npmjs.com/package/prop-types) added to package.json for [typechecking props](https://reactjs.org/docs/typechecking-with-proptypes.html) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/section/propTypes.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+👤 **Fares Ashraf** 
+
+[![](https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white)](https://github.com/ashraffares) [![](https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/Fares09301164) [![](https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/faresashraf/)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ashraffares/Phaser_Game/issues).
```
Comment (rindrajosia) [inline README.md]: - [ ] [REQUIRED] Please make sure that this URL (issues pages) redirects to the correct repo.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [ ] [REQUIRED] app should be deployed on Heroku(or other hosts like Netlify, Firebase, etc) and the live linkshould be  included at the README file.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/app-deployed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by bellom — APPROVED** [review]
## Hi @ashraffares  :clap: 

**Great work making all the necessary corrections from the previous review.**

# Status: Approved 👍 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit the project completion form.
Good luck in the next milestone :slightly_smiling_face:_ 

**Heads-Up:** _Make sure to read the instructions on pathwright carefully before submitting CRR for milestone 2, that way, you will have a high chance of getting your CRR reviewed just once without submitting it for re-review_


**Best**

**Pull request merged**